src/main/scala/Dromajo.scala

```diff
@@ -0,0 +1,101 @@
+package testchipip
+
+import chisel3._
+import chisel3.util._
+import chisel3.experimental.{IntParam, StringParam}
+import freechips.rocketchip.config.{Parameters}
+
+/**
+ * Dromajo bridge to input instruction streams and check with Dromajo
+ */
+class SimDromajoBridge(insnWidths: TracedInstructionWidths, numInsns: Int) extends Module
+{
+  val io = IO(new Bundle {
+    val trace = Input(new TileTraceIO(insnWidths, numInsns))
+  })
+
+  // constants
+  val xLen = 64
+  val instBits = 32
```

Comment:
This is a requirement of dromajo? Does it get checked against `insnWidths`?

Reply:
Yes. Dromajo is RV64GC. It doesn't check against `insnWidths`